src/crimson/osd/pg.cc

```diff
@@ -677,7 +677,7 @@ PG::interruptible_future<Ref<MOSDOpReply>> PG::handle_failed_op(
   return (need_reload_obc ? reload_obc(*obc)
                           : interruptor::make_interruptible(load_obc_ertr::now())
   ).safe_then_interruptible([e, &m, obc = std::move(obc), this] {
-    auto reply = make_message<MOSDOpReply>(
+    auto reply = ceph::make_message<MOSDOpReply>(
```

Comment:
Why `ceph::` and not `crimson::`?

Reply:
I see, you want to convert this method (btw, same applies to `OSD::get_stats()`) as a follow-up. I'm fine with that.